CONTRIBUTING.md

```diff
@@ -58,7 +58,7 @@ While the prerequisites above must be satisfied prior to having your pull reques
 
 ### Reporting Bugs
 
-This section guides you through submitting a bug report for Atala PRISM KMM SDK. Following these guidelines helps maintainers and the community understand your report :pencil:, reproduce the behaviour :computer: :computer:, and find related reports :mag_right:.
+This section guides you through submitting a bug report for Identus KMM SDK. Following these guidelines helps maintainers and the community understand your report :pencil:, reproduce the behaviour :computer: :computer:, and find related reports :mag_right:.
```

Comment:
`Identus KMM SDK` should be `Edge Agent SDK KMM` according to the [replacement table](https://docs.google.com/spreadsheets/d/1lR3abdg6qExXasUVqBlFouNu9FCB0I1ynMMZodQ6y8U/edit#gid=0)